src/data/eventregistration_de.json

```diff
@@ -50,7 +50,7 @@
                 "wrongfileheaders": "Die Header der .csv-Datei entsprechen nicht den Headern der entsprechenden Vorlage, Sie können sie <a href='. /event-qr-code-guide/'>hier überprüfen.</a>",
                 "wrongfileformatfields": "Einige Felder in Ihrer CSV-Datei sind nicht korrekt ausgefüllt. Bitte überprüfen Sie Ihre .csv und versuchen Sie es erneut",
                 "filewithnodata": "Ihre .csv Felder sind nicht ausgefüllt",
-                "fileoverloaded": "Die Erstellung von QR-Codes wurde auf 100 . begrenzt"
+                "fileoverloaded": "Die Erstellung von QR-Codes wurde auf 100 begrenzt."
```

Comment:
Wouldn't it make more sense to write "Die Erstellung von QR-Codes ist auf 100 begrenzt." "wurde begrenzt" means that it was possible for some time to create more than 100 QR codes but now it was limited to 100. This is not the case here. 
Same in English.

Reply:
fixed in this commit https://github.com/corona-warn-app/cwa-website/pull/2105/commits/c56895e8cbcccada09a59da130e0423820a908d0

Reply:
alright, you are right, it is about the tense.

Reply:
Thank you @dsarkar!

Reply:
thanks for review

Reply:
You're very welcome!